packages/language-server/src/plugins/typescript/features/DiagnosticsProvider.ts

```diff
@@ -5,8 +5,22 @@ import { DiagnosticsProvider } from '../../interfaces';
 import { LSAndTSDocResolver } from '../LSAndTSDocResolver';
 import { convertRange, getDiagnosticTag, mapSeverity } from '../utils';
 import { SvelteDocumentSnapshot, SvelteSnapshotFragment } from '../DocumentSnapshot';
-import { isInGeneratedCode, isAfterSvelte2TsxPropsReturn } from './utils';
-import { regexIndexOf, swapRangeStartEndIfNecessary } from '../../../utils';
+import {
+    isInGeneratedCode,
+    isAfterSvelte2TsxPropsReturn,
+    findNodeAtSpan,
+    isReactiveStatement,
+    isInReactiveStatement,
+    gatherIdentifiers
+} from './utils';
+import { not, flatten, passMap, regexIndexOf, swapRangeStartEndIfNecessary } from '../../../utils';
+
+enum DiagnosticCode {
+    // "Left side of comma operator is unused and has no side effects."
+    DIAGNOSTIC_CODE_NOOP_IN_COMMAS = 2695,
+    // "Unused label."
+    DIAGNOSTIC_CODE_UNUSED_LABEL = 7028
+}
```

Comment:
Really like the enum idea! Two suggestions:
1. remove the `DIAGNOSTIC_CODE_` prefix, that's already apparant from using the enum
2. Could you add the other diagnostic codes that are used in this file, too?

Reply:
Sure.